eng/Version.Details.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Dependencies>
+  <ProductDependencies>
```

Comment:
Is there a reason we're not taking CoreFX / CoreCLR as dependencies here?

Reply:
This is just first step to get us plugged in. Once we have this working we can add add other dependencies.

Reply:
I'd leave that to you :)